### PR TITLE
A more detailed description has been added.

### PR DESCRIPTION
**Pull request by Fabian-t800**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,43 @@ Test Spotify API endpoints, using POSTMAN.
...
+		- Note: Scopes are very important when retreiving your authorization code. Scopes will define what you can and what you cannot do later on and will affect the running of test suites.
+		- Note: you can find a <a href="https://developer.spotify.com/documentation/general/guides/scopes/" =>list of scopes in this link here </a>.
+	
 Steps needed to generate the refresher token for a new user can be found here (2nd paragraph Have your application request refresh and access tokens; Spotify returns access and refresh tokens):
 	- https://developer.spotify.com/documentation/general/guides/authorization-guide/
 	
+
+------------------------------
+
+Explenation of environment variables
```
Comment (nicolae-chedea) [inline README.md]: eplenation should be replaced with explanation

File: README.md [hunk 2]
```diff
@@ -3,21 +3,43 @@ Test Spotify API endpoints, using POSTMAN.
...
+	An explenation of all the environment variables will be available below:
+	+ "user_id" - the user ID you're looking for. Initially it would be recomended that you give your own user id
+	+ "api_domain" - the API endpoint domain. By default it should be: "https://api.spotify.com"
+	To be continued.
```
Comment (nicolae-chedea) [inline README.md]: add some steps to describe how to use environment variables in Postman, that is really helpful for someone who is not an expert in Postman

**Review by nicolae-chedea — CHANGES_REQUESTED** [review]

**Review by DanDaika — APPROVED** [review]
Made all the changes needed after the first DEMO

File: README.md (hunk 2 above)
Comment (DanDaika) [reply README.md]: Added extra steps

**Review by DanDaika — APPROVED** [review]
Updated

**Review by DanDaika — APPROVED** [review]

**Review by DanDaika — APPROVED** [review]

**Review by nicolae-chedea — APPROVED** [review]

**Pull request merged**